src/Tizen.NUI/src/public/WidgetViewManager.cs

```diff
@@ -55,6 +55,18 @@ public WidgetView AddWidget(string widgetId, string contentInfo, int width, int
             return ret;
         }
 
+        /// <summary>
+        /// Remove a widget view object.
+        /// </summary>
+        /// <param name="widgetView">widgetView to remove</param>
+        /// <returns> True on success, false otherwise.</returns>
+        public bool RemoveWidget(WidgetView widgetView)
```

Comment:
The public API changes should be processed in master branch only.
So, make API as hidden by adding [EditorBrowsable(EditorBrowsableState.Never)] attribute, then in DevelNUI branch API-Checker will pass.
After the patch is merged into master branch by regular integration work, you can make new PR in master branch by removing [EditorBrowsable(EditorBrowsableState.Never)] attribute to proceed ACR.
if the ACR is urgent, you can upload PR on master branch directly. that PR will be merged on DevelNUI branch by rebasing during the regular integration work.